data/json/items/software.json

```diff
@@ -70,5 +70,14 @@
     "name_plural": "train data",
     "description": "Logistical data on subterranean train routes and schedules.",
     "price": 200
+  },
+  {
+    "id": "mind_scan_robofac",
+    "type": "GENERIC",
+    "name": "neural data",
+    "symbol": "#",
+    "container": "data_card",
+    "description": "Data stolen from a dead scientist memory banks.  Is the owner of these thoughts still hidden here, amidst the unreadable data; or are these just a collection of the precious moments of someone's life?\n\nWhatever the case, the idea of perpetually keeping a part of you within a metallic pill makes you uncomfortable.",
```

Comment:
```suggestion
    "description": "Data stolen from a dead scientist memory banks.  Is the owner of these thoughts still hidden here, amidst the unreadable data, or are these just a collection of the precious moments of someone's life?\n\nWhatever the case, the idea of perpetually keeping a part of you within a metallic pill makes you feel uncomfortable.",
```

Reply:
Its intended but Im aware it breaks normal conventions.